components/hab-butterfly/src/main.rs

```diff
@@ -33,24 +35,32 @@ use clap::ArgMatches;
 use common::ui::{Coloring, UI, NOCOLORING_ENVVAR, NONINTERACTIVE_ENVVAR};
 use hcore::env as henv;
 use hcore::crypto::{init, default_cache_key_path, BoxKeyPair, SymKey};
-use hcore::fs::FS_ROOT_PATH;
 use hcore::service::ServiceGroup;
 
 use hab_butterfly::{analytics, cli, command};
 use hab_butterfly::error::{Error, Result};
 
-
 /// Makes the --org CLI param optional when this env var is set
 const HABITAT_ORG_ENVVAR: &'static str = "HAB_ORG";
 /// Makes the --user CLI param optional when this env var is set
 const HABITAT_USER_ENVVAR: &'static str = "HAB_USER";
-
-const FS_ROOT_ENVVAR: &'static str = "FS_ROOT";
-
 const HABITAT_BUTTERFLY_PORT: u64 = 9638;
-
 const MAX_FILE_UPLOAD_SIZE_BYTES: u64 = 4096;
 
+lazy_static! {
+    /// The default filesystem root path to base all commands from. This is lazily generated on
+    /// first call and reflects on the presence and value of the environment variable keyed as
+    /// `FS_ROOT_ENVVAR`.
+    static ref FS_ROOT: PathBuf = {
```

Comment:
Probably could have been a simple function as the subcommands only ever get invoked once but this works for me.